sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj

```diff
@@ -4,13 +4,14 @@
 	<!-- Please do not move/edit code below this line -->
 
```

Comment:
Should have a CHANGELOG under this folder, like this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Microsoft.Azure.Management.EventGrid/CHANGELOG.md

Reply:
Added change.log file with the release information.